.github/workflows/makefile-test.yaml

```diff
@@ -0,0 +1,31 @@
+name: Use FreeRTOS/coreHTTP proofs to test CBMC starter-kit 
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, labeled, unlabeled]
+  push:
+    branches: [ master ]
+
+jobs:
+  run-tests:
+    if: "!contains(github.event.pull_request.labels.*.name, 'no-test')"
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [ '3.8', '3.9', '3.10' ]
+    name: Python ${{ matrix.python-version }}
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python
+        uses: actions/setup-python@v3
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          brew install scdoc mandoc coreutils ninja
```

Comment:
Since you are running it on `ubuntu-latest`, I would suggest installing dependencies with `apt` and not `brew` even though it works either way in GitHub Actions.

Reply:
I decided to go with `brew` because almost all dependencies can be installed within one command. As you suggested `apt` could indeed work as well.